mappings/net/minecraft/util/math/BlockPos.mapping

```diff
@@ -151,6 +194,11 @@ CLASS net/minecraft/class_2338 net/minecraft/util/math/BlockPos
 		FIELD field_23101 limitY I
 		FIELD field_23102 dx I
 		FIELD field_23103 dy I
+		FIELD field_23375 centerZ I
+		FIELD field_23376 centerX I
+		FIELD field_23377 centerY I
+		FIELD field_23378 blockPos Lnet/minecraft/class_2338$class_2339;
```

Comment:
should be an opposite pos from the current calculated one

Reply:
what do you mean?

Reply:
if i recall right this should be an iterator that iterates outwards, isn't it?

Reply:
Yes?

Reply:
The outward iterator is weird. It calculates a pair of positions at once, and holds one in a field so it would be returned in the next `next` call. if this is the block pos cached for the next `next` call imo `oppositeNext` is better; if this is the pos passed as the original center then `center` is better